jacoco/pom.xml

```diff
@@ -111,8 +111,8 @@
             <configuration>
               <rules>
                 <requireFilesSize>
-                  <maxsize>4600000</maxsize>
-                  <minsize>4000000</minsize>
+                  <maxsize>4700000</maxsize>
+                  <minsize>4100000</minsize>
```

Comment:
even without this PR we got close to upper limit - see https://github.com/jacoco/jacoco/commit/5334d9fb6cbcd78e35217556a40703b927c2b80f